.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Tests
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    name: Run on Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone the code
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: '~1.22'
+
+      - name: Running Tests
+        run: |
+          go mod tidy
```

Comment:
I think I don't understand the false positives part, as IMO this just cleans the go.sum file. My guess is that this is done just to prevent downloading unused dependencies.

In any case, I do agree that is a good practice to ensure that `go.sum` is always clean. I'll add the check.

Reply:
I added the check.